src/views-components/process-remove-dialog/process-remove-dialog.tsx

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) The Arvados Authors. All rights reserved.
+//
+// SPDX-License-Identifier: AGPL-3.0
+
+import { Dispatch } from "redux";
+import { connect } from "react-redux";
+import { ConfirmationDialog } from "~/components/confirmation-dialog/confirmation-dialog";
+import { withDialog, WithDialogProps } from "~/store/dialog/with-dialog";
+import { removeProcessPermanently, REMOVE_PROCESS_DIALOG } from "~/store/processes/processes-actions";
+
+ const mapDispatchToProps = (dispatch: Dispatch, props: WithDialogProps<any>) => ({
+    onConfirm: () => {
+        props.closeDialog();
+        dispatch<any>(removeProcessPermanently(props.data.uuid));
+    }
+});
+
+ export const [RemoveProcessDialog] = [ConfirmationDialog]
```

Comment:
Could you replace the construction with `map` with `compose` just like in other places?